website/src/e-commerce/src/widgets/PriceSlider.js

```diff
@@ -0,0 +1,13 @@
+import { panel, rangeSlider } from 'instantsearch.js/es/widgets';
+
+// The price range slider has a dummy panel to be collapsed
+// at the same time as `rangeInput`.
+const priceRangeSlider = panel({})(rangeSlider);
```

Comment:
As you can see in the [design](https://app.zeplin.io/project/5acc8becfc0c980068fc12cc/screen/5cd1a7992e4f02039b5a8336), the `rangeInput` is wrapped in the same panel as `rangeSlider`. However, InstantSearch.js only supports wrapping a single widget in a panel.

This will be required once the styles come.